src/components/Auth/InputCustom.tsx

```diff
@@ -0,0 +1,94 @@
+export interface InputCustomProps {
+  formValue: string
+  label: string
+  placeholder?: string
+  type?: string
+  required?: boolean
+  validation?: any
+  messageError?: string
+}
+import {
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  Icon,
+  Input,
+  InputGroup,
+  InputRightElement,
+  Text,
+} from '@chakra-ui/react'
+import { useState } from 'react'
+import { useFormContext } from 'react-hook-form'
+import { useTranslation } from 'react-i18next'
+import { MdOutlineRemoveRedEye } from 'react-icons/md'
+import { RiEyeCloseLine } from 'react-icons/ri'
+import useDarkMode from '~src/themes/saas/hooks/useDarkMode'
+
+export interface InputCustomProps {
+  formValue: string
+  label: string
+  placeholder?: string
+  type?: string
+  required?: boolean
+  validation?: any
+  messageError?: string
+}
```

Comment:
You should add the chakra props for the FormLabel and Input so you will avoid to add type placeholder and others... Something like:

```ts
export interface InputCustomProps {
  formValue: string
  label: string
  required?: boolean
  validation?: any
  messageError?: string
  labelProps:  FormLabelProps
} & InputProps
```

(note that you don't need to declare `placeholder` or `type` properties because they are already defined on the chakra `InputProps`)